### PR TITLE
PigTimer

### PR DESCRIPTION
**Pull request by ewjax**
Reworked the custom timer interface.

All below commands work in tells (keep them private!) or regular say (share with your friends!)

      /t PigTimer-duration
      /t PigTimer-duration-label
 or 
      /say PigTimer-duration
      /say PigTimer-duration-label

 where
 
      "PigTimer" is the identifying marker
      "duration" can be in time formats:
           hh:mm:ss
           mm:ss
           seconds
      "label" can contain any character, number, or underscore, but cannot have blank spaces

      "PigTimer", "duration", and "label" should be separated by a - dash

Examples:

      PigTimer-30                                30 second timer, with no label
      PigTimer-10-TenSeconds            10 second timer, with label
      PigTimer-10:00-TenMinutes        10 minute timer, with label
      PigTimer-10:00:00-TenHours       10 hour timer, with label
      PigTimer-120-description          120 second timer, with label 'description'
      PigTimer-6:40-Guard_George        6 minutes 40 second timer, with label 'Guard_George'
      PigTimer-1:02:00-LongTimer         1 hour, 2 minute timer, with label 'LongTimer'

The selection of "PigTimer" as the marker is in keeping with the whole PigParse theme, but is obviously somewhat arbitrary, and could be literally anything else.  It could also by synonym'd, like (PigTimer|.pt), for example.

### CODE REVIEW

File: EQTool/Services/Parsing/LogStartCustomTimer.cs [hunk 1]
```diff
@@ -27,61 +56,70 @@ public bool Handle(string line, DateTime timestamp)
...
+            // This website is absolutely needed when creating this kind of crazy regex!
+            //      https://regex101.com/r/RWA9Oy/1
+            Regex regex = new Regex(customTimerPattern, RegexOptions.Compiled);
+            var match = regex.Match(line);
+            if (match.Success)
             {
-                timer = GetStartTimer(message, StartTimer1);
-            }
+                // just a little audible marker
+                System.Media.SoundPlayer player = new System.Media.SoundPlayer(@"c:\Windows\Media\chimes.wav");
```
Comment (smasherprog) [inline EQTool/Services/Parsing/LogStartCustomTimer.cs:88]: Please remove the playing of sound in the parsers. It makes it so when running tests locally, it actually plays sounds.
Also, it means that everyone is going to hear these sounds every time its parsed.
If we want a sounds to play when this is parse happens, a class that listens for these events can play that logic

**Review by smasherprog — COMMENTED** [review]
Change this comment. Breaking change is fine for this

**Comment by smasherprog** [comment]
Other than the comments above, this is a good change!

File: EQTool/Services/Parsing/LogStartCustomTimer.cs (hunk 1 above)
Comment (ewjax) [reply EQTool/Services/Parsing/LogStartCustomTimer.cs:88]: can do.  that's really just a debugging help for me, so I know the parser is doing it's thing

**Pull request merged**